src/librustdoc/html/render/cache.rs

```diff
@@ -50,7 +52,7 @@ crate fn extern_location(
 
     // Failing that, see if there's an attribute specifying where to find this
     // external crate
-    e.attrs
+    ast_attrs
```

Comment:
It looks like this was the only place the attributes for ExternalCrate was used, so you could turn it into a parameter instead - that's a great change, but it makes it a little harder to see what changes are related to removing files from Attributes itself. Do you mind splitting it into a separate commit?

Reply:
What should I do here ? Revert the changes on `ExternalCrate` and put them in a separate MR ?

Also I didn't really get the "turn it into a parameter instead" part. Can you give me a little bit more explanation ?

Reply:
> Should I revert the changes on ExternalCrate and put them in a separate MR ?

Nah, don't worry about it, this is fine :) It's a good change.

> Also I didn't really get the "turn it into a parameter instead" part. Can you give me a little bit more explanation ?

This is something you're already doing: instead of storing the attributes on ExternalCrate, you pass them in to `extern_location`, which is the only place that uses them.